docker/test/base/Dockerfile

```diff
@@ -33,6 +33,9 @@ ENV TSAN_OPTIONS='halt_on_error=1 abort_on_error=1 history_size=7 memory_limit_m
 ENV UBSAN_OPTIONS='print_stacktrace=1'
 ENV MSAN_OPTIONS='abort_on_error=1 poison_in_dtor=1'
 
+# for external_symbolizer_path
```

Comment:
wait, it's build from `clickhouse/test-util`, and fasttests are built from there as well. I thought, fasttests need it too? Or not?

Reply:
fasttest don't run tests with sanitizers, plus all other settings are in test/base anyway.